### PR TITLE
Feature/add aerial imagery

### PR DESCRIPTION
**Pull request by Outtascope**
Added aerial imagery and a menu option to toggle its display.

### CODE REVIEW

File: app/src/main/java/com/example/android/searchabledict/MapDisplay.java [hunk 1]
```diff
@@ -55,6 +55,7 @@
 import com.esri.core.tasks.na.StopGraphic;
 import com.google.maps.android.geometry.Bounds;
 import com.google.maps.android.quadtree.PointQuadTree;
+import com.esri.android.map.ags.ArcGISDynamicMapServiceLayer;
```
Comment (Outtascope) [inline app/src/main/java/com/example/android/searchabledict/MapDisplay.java:58]: Imagery comes from a dynamic service layer

File: app/src/main/java/com/example/android/searchabledict/MapDisplay.java [hunk 2]
```diff
@@ -136,6 +138,8 @@ protected void onCreate(Bundle savedInstanceState) {
 
         // Retrieve the map and initial extent from XML layout
         mMap = (MapView) findViewById(R.id.map);
+        aerialLayer = new ArcGISDynamicMapServiceLayer("http://prod.gis.msu.edu/arcgis/rest/services/basemap/aerial/MapServer");
+        mMap.addLayer(aerialLayer);
```
Comment (Outtascope) [inline app/src/main/java/com/example/android/searchabledict/MapDisplay.java:142]: Add it first so it is at the bottom of the display stack.

File: app/src/main/java/com/example/android/searchabledict/MapDisplay.java [hunk 3]
```diff
@@ -206,6 +211,10 @@ public void onClick(View v) {
         locationDisplayManager.setAutoPanMode(LocationDisplayManager.AutoPanMode.OFF);
     }
 
+    public void toggleSatellite() {
+        aerialLayer.setVisible(!aerialLayer.isVisible());
+    }
+
```
Comment (Outtascope) [inline app/src/main/java/com/example/android/searchabledict/MapDisplay.java:217]: Just flip the visibility state in response to a toggle request.

File: app/src/main/java/com/example/android/searchabledict/MapDisplay.java [hunk 4]
```diff
@@ -613,6 +622,10 @@ public boolean onOptionsItemSelected(MenuItem item) {
                 startActivity(searchIntent);
                 return true;
 
+            case R.id.toggleAerial:
+                toggleSatellite();
+                return true;
+
```
Comment (Outtascope) [inline app/src/main/java/com/example/android/searchabledict/MapDisplay.java:628]: Catch the toggle menu events.

File: app/src/main/res/menu/menu.xml [hunk 5]
```diff
@@ -7,6 +7,8 @@
 
 	<item android:id="@+id/clearPaths"     
     	android:title="Clear Path" />
+	<item android:id="@+id/toggleAerial"
+		android:title="Toggle Aerial Imagery"/>
```
Comment (Outtascope) [inline app/src/main/res/menu/menu.xml:12]: Our menu option.

**Review by wagenma9 — APPROVED** [review]

**Pull request merged**